dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/persistence/file/batch/BatchFileOrchestrator.kt

```diff
@@ -67,6 +77,7 @@ internal class BatchFileOrchestrator(
         }
 
         deleteObsoleteFiles()
+        lastCleanupTimestamp = System.currentTimeMillis()
```

Comment:
maybe the update of `lastCleanupTimestamp` can be part of the `deleteObsoleteFiles()` method?